From de754a3d48b419731f76841850cc7f2e872c2281 Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Tue, 5 Sep 2023 13:14:04 +0200
Subject: [PATCH] m4: do not force -I/usr/include in CPPFLAGS

This is so horribly broken for cross-compilation. :-(

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
[rebased against stable-12.3.0]
Signed-off-by: Stefan Agner <stefan@agner.ch>
---
 m4/vmtools.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/m4/vmtools.m4 b/m4/vmtools.m4
index 055b39d..d281f81 100644
--- a/m4/vmtools.m4
+++ b/m4/vmtools.m4
@@ -279,10 +279,10 @@ AC_DEFUN([AC_VMW_DEFAULT_FLAGS],[
       if test "$os" = freebsd; then
          CUSTOM_$1_CPPFLAGS="-I/usr/local/include"
       else
-         CUSTOM_$1_CPPFLAGS="-I/usr/include"
+         CUSTOM_$1_CPPFLAGS=" "
       fi
       if test -n "$2"; then
-         CUSTOM_$1_CPPFLAGS="${CUSTOM_$1_CPPFLAGS}/$2"
+         : CUSTOM_$1_CPPFLAGS="${CUSTOM_$1_CPPFLAGS}/$2"
       fi
    fi
 ])
